x/datadeal/types/datasale.go

```diff
@@ -77,3 +77,27 @@ func (m DataSale) ValidateBasic() error {
 
 	return nil
 }
+
+func (m DataDeliveryVote) ValidateBasic() error {
+	if m.DealId <= 0 {
```

Comment:
It seems to be enough to check if the value is zero or not because the type of `DealId` is `uint64` :)

Reply:
Thanks, I fix it.